### DISCUSSION TITLE
Batch Process for Prompt Input

### DISCUSSION THREAD

**Discussion by mrayeung** [lllyasviel/Fooocus#2474] [Q&A] [answered]
Hi. I'm a new to Fooocus AI. I think it is a great interface for Stable Diffusion. I wanted to confirm whether this is a way to run batch process for Fooocus. I'm using nVidia GPU with Ubuntu in a local computer.  Let say I want to generate 1000 images with 100 different prompts. Is there a way to feed the list of prompts to Fooocus in the UI or using a script?'

Any feedback is appreciated. Thanks!

**Comment by mashb1t** [comment, marked as answer]
Yes, but only after PR https://github.com/lllyasviel/Fooocus/pull/1761 (use wildcard file) is merged or with API.
